src/components/NavBar/SearchBar.tsx

```diff
@@ -58,6 +58,7 @@ export const SearchBar = () => {
       refetchOnWindowFocus: false,
       refetchOnMount: false,
       refetchOnReconnect: false,
+      enabled: !!debouncedSearchValue,
```

Comment:
you might need to condition based off length, I'm still able to fire off a blank search. 
Steps:
- enter in a character(including space)
- delete character
- In network tab a blank search fires off